apps/web/src/components/score/Score.tsx

```diff
@@ -0,0 +1,71 @@
+import { Heading, Flex, VStack } from '@chakra-ui/react';
+import { useSession } from 'next-auth/react';
+import { useEffect, useState } from 'react';
+import { UserScore } from 'types/Score';
+
+export const Score = () => {
+  const { data: session } = useSession();
+  const [errors, setErrors] = useState<Record<string, unknown> | boolean>(
+    false,
+  );
+  const [loading, setLoading] = useState<boolean>(true);
+  const [score, setScore] = useState<UserScore>({
+    retweet_count: 0,
+    reply_count: 0,
+    like_count: 0,
+    quote_count: 0,
+    total: 0,
+  });
+
+  useEffect(() => {
+    if (session) {
+      const updateScore = async (session: any) => {
+        const body = new URLSearchParams({
+          providerAccountId: session?.id,
```

Comment:
can't we consume this data in the backend?

can't we use `useSession` in the backend?